my/time/tz/common.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from typing import Literal, Callable
+
+from ...core.common import tzdatetime
+
+
+'''
+Depending on the specific data provider and your level of paranoia you might expect different behaviour.. E.g.:
+- if your objects already have tz info, you might not need to call localize() at all
+- it's safer when either all of your objects are tz aware or all are tz unware, not a mixture
+- you might trust your original tiemzone, or it might just be UTC, and you want to use something more reasonable
+'''
+Policy = Literal[
+    'keep'   , # if datetime is tz aware, just preserve it
+    'convert', # if datetime is tz aware, convert to provider's tz
+    'throw'  , # if datetime is tz aware, throw exception
+    # todo 'warn'? not sure if very useful
+]
+# todo make the default configurable, e.g. add my.cofnig.time.tz section
+DEFAULT_POLICY: Policy = 'keep'
```

Comment:
not sure what the default should be, 'keep' or 'convert'? @seanbreckenridge what do you think? 
I'm not sure because I'm expecting to use this in 'error' mode most of the time to fail fast :) 
An upside of `convert` would be is that you basically put trust into the `tz` module, which is ultimately good. A downside is that it might be too coarse (e.g. if `tz` fell back on the `home` locations), in that case the original timezone might be better.

Reply:
I assume this will will be configurable through `my.config`. I think it makes sense to have the default be `keep`, forcing the user to be aware of the `tz` module and configure it makes changes to the data they may not be expecting.